runelite-client/src/main/java/net/runelite/client/ui/FontManager.java

```diff
@@ -36,37 +39,53 @@
 	private static final Font runescapeSmallFont;
 	private static final Font runescapeBoldFont;
 
+	// By using a 2way map we're not dependant on the toString() method so we have control over the font naming in the UI.
+	private static final HashMap<String, Font> fontLookupMap;
+	// By using a linkedHashMap we decide the order in which the fonts a re represented which allows more intuitive ordering, i couldn't get ordering to swork with guave bimaps
+	private static final LinkedHashMap<Font, String> fontMap;
+
 	static
 	{
 		GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
 
 		try
 		{
-			Font font = Font.createFont(Font.TRUETYPE_FONT,
+			runescapeFont = Font.createFont(Font.TRUETYPE_FONT,
 				FontManager.class.getResourceAsStream("runescape.ttf"))
 				.deriveFont(Font.PLAIN, 16);
-			ge.registerFont(font);
-
-			runescapeFont = StyleContext.getDefaultStyleContext()
-					.getFont(font.getName(), Font.PLAIN, 16);
-			ge.registerFont(runescapeFont);
 
-			Font smallFont = Font.createFont(Font.TRUETYPE_FONT,
+			runescapeSmallFont = Font.createFont(Font.TRUETYPE_FONT,
 				FontManager.class.getResourceAsStream("runescape_small.ttf"))
 				.deriveFont(Font.PLAIN, 16);
-			ge.registerFont(smallFont);
-
-			runescapeSmallFont = StyleContext.getDefaultStyleContext()
-					.getFont(smallFont.getName(), Font.PLAIN, 16);
-			ge.registerFont(runescapeSmallFont);
 
-			Font boldFont = Font.createFont(Font.TRUETYPE_FONT,
+			runescapeBoldFont = Font.createFont(Font.TRUETYPE_FONT,
 					FontManager.class.getResourceAsStream("runescape_bold.ttf"))
 					.deriveFont(Font.PLAIN, 16);
-			ge.registerFont(boldFont);
 
-			runescapeBoldFont = StyleContext.getDefaultStyleContext()
-					.getFont(boldFont.getName(), Font.PLAIN, 16);
+			fontMap = new LinkedHashMap<>();
+			fontLookupMap = new HashMap<>();
+
+			fontMap.put(runescapeSmallFont, "rs small");
```

Comment:
Why are these short abbreviations used? just use normal name

Reply:
Mainly because of fitting inside the drop down but alright boss